src/components/FirstBanner/FirstBanner.style.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
+
+export const Title = styled.h1`
+  font-size: 48px;
```

Comment:
Add styles from theme so that we can standardize it for font-sizes, colors, spacing and radius